PKHeX.Core/Legality/Core.cs

```diff
@@ -2085,8 +2085,11 @@ internal static bool IsFormChangeable(PKM pkm, int species)
                 return true;
             if (IsEvolvedFormChange(pkm))
                 return true;
-            if (pkm.Species == 718 && pkm.InhabitedGeneration(7) && pkm.AltForm == 3)
```

Comment:
The interior check is basically `return pkm.AltForm != 1`.
It would be more succinct to replace `pkm.AltForm == 3` with `pkm.AltForm != 1`; same result but less lines.